README.md

```diff
@@ -1,2 +1,2 @@
-# template-repository-conference
-template repository for conference proceedings
+# Computational geophysics in a changing climate 
+
```

Comment:
@tetikshajain: would you mind also putting the abstract in the README? (we can do this for all presentations) Thanks!